tests/parser/functions/test_unsafe_math.py

```diff
@@ -52,3 +52,45 @@ def foo(x: {typ}, y: {typ}) -> {typ}:
         ys += [0, 1, hi - 1, hi, fixed_pt]
         for (x, y) in itertools.product(xs, ys):
             assert c.foo(x, y) == fn(x, y) % mod_bound
+
+
+@pytest.mark.parametrize("typ", integer_types)
+@pytest.mark.parametrize("literal", [-2, -1, 1, 2])
+@pytest.mark.parametrize("op", ["add", "sub", "mul", "div"])
+@pytest.mark.fuzzing
+def test_unsafe_op_literal(get_contract, typ, literal, op):
+    code = f"""
+@external
+def foo(x: {typ}) -> {typ}:
+    return unsafe_{op}(x, {literal})
```

Comment:
seems like some code duplication with the other `test_unsafe_op` function. is it possible to merge the logic of these two?